backend/src/schemas/slide-object.schema.ts

```diff
@@ -0,0 +1,227 @@
+
+  @Prop({ required: true })
+  strokeDashOffset: number;
+
+  @Prop({ required: true, enum: ['miter', 'round', 'bevel'] })
+  strokeLineJoin: 'miter' | 'round' | 'bevel';
+
+  @Prop({ required: true })
+  strokeUniform: boolean;
+
+  @Prop({ required: true })
+  strokeMiterLimit: number;
+
+  @Prop({ required: true })
+  scaleX: number;
+
+  @Prop({ required: true })
+  scaleY: number;
+
+  @Prop({ required: true })
+  angle: number;
+
+  @Prop({ required: true })
+  flipX: boolean;
+
+  @Prop({ required: true })
+  flipY: boolean;
+
+  @Prop({ required: true })
+  opacity: number;
+
+  @Prop({ type: Shadow, default: null })
+  shadow: Shadow | null;
+
+  @Prop({ required: true })
+  visible: boolean;
+
+  @Prop({ required: true })
+  backgroundColor: string;
+
+  @Prop({ required: true })
+  fillRule: string;
+
+  @Prop({ required: true, enum: ['fill', 'stroke'] })
+  paintFirst: 'fill' | 'stroke';
+
+  @Prop({ required: true })
+  globalCompositeOperation: string;
+
+  @Prop({ required: true })
+  skewX: number;
+
+  @Prop({ required: true })
+  skewY: number;
+
+  @Prop({
+    type: [[String, Number, Number]],
+    required: true,
+  })
+  path: Array<[string, number, number]>;
+
+  @Prop({ type: Number })
+  x1?: number;
+
+  @Prop({ type: Number })
+  x2?: number;
+
+  @Prop({ type: Number })
+  y1?: number;
+
+  @Prop({ type: Number })
+  y2?: number;
+
+  @Prop({ type: Number })
+  rx?: number;
+
+  @Prop({ type: Number })
+  ry?: number;
+
+  @Prop({ type: Number })
+  radius?: number;
+
+  @Prop({ type: Number })
+  startAngle?: number;
+
+  @Prop({ type: Number })
+  endAngle?: number;
+
+  @Prop({ type: String })
+  fontFamily?: string;
+
+  @Prop({ type: String })
+  fontWeight?: string;
+
+  @Prop({ type: Number })
+  fontSize?: number;
+
+  @Prop({ type: String })
+  text?: string;
+
+  @Prop({ type: Boolean })
+  underline?: boolean;
+
+  @Prop({ type: Boolean })
+  overline?: boolean;
+
+  @Prop({ type: Boolean })
+  linethrough?: boolean;
+
+  @Prop({ enum: ['left', 'center', 'right'] })
+  textAlign?: 'left' | 'center' | 'right';
+
+  @Prop({ enum: ['normal', 'italic'] })
+  fontStyle?: 'normal' | 'italic';
+
+  @Prop({ type: Number })
+  lineHeight?: number;
+
+  @Prop({ type: String })
+  textBackgroundColor?: string;
+
+  @Prop({ type: Number })
+  charSpacing?: number;
+
+  @Prop({ type: mongoose.Schema.Types.Mixed })
+  styles?: {
+    [key: string]: {
+      textDecoration: string;
+    };
+  }[];
+
+  @Prop({ enum: ['ltr', 'rtl'] })
+  direction?: 'ltr' | 'rtl';
+
+  @Prop({ type: Number })
+  pathStartOffset?: number;
+
+  @Prop({ enum: ['left', 'right', 'center'] })
+  pathSide?: 'left' | 'right' | 'center';
+
+  @Prop({ enum: ['baseline', 'middle', 'top', 'bottom'] })
+  pathAlign?: 'baseline' | 'middle' | 'top' | 'bottom';
+
+  @Prop({ type: String })
+  src?: string;
+
+  @Prop({ type: Number })
+  cropX?: number;
+
+  @Prop({ type: Number })
+  cropY?: number;
+
+  @Prop({
```

Comment:
code reformat
![image](https://github.com/user-attachments/assets/2451e22c-5884-46b5-a0cf-b833276719ac)

Reply:
Previous version was more readable for me, besides we wen from 230 to 360 lines. But whatever.